ocaml/libs/tracing/tracing.mli

```diff
@@ -76,10 +78,20 @@ module Tracer : sig
   val span_is_finished : Span.t option -> bool
 
   val span_hashtbl_is_empty : unit -> bool
+
+  val finished_span_hashtbl_is_empty : unit -> bool
 end
 
 module TracerProvider : sig
-  type t
+  type t = {
+      name_label: string
+    ; tags: (string * string) list
+    ; endpoints: endpoint list
+    ; filters: string list
+    ; processors: string list
+    ; enabled: bool
+    ; service_name: string
```

Comment:
This needs documentation. What is the difference to `name_label`?

Reply:
service_name refers  to a component name i.e. xapi and is responsible for the difference in bar colours seen in the UI. name_label is a user defined label that is used to identify a provider